session/txn.go

```diff
@@ -455,6 +461,9 @@ func (s *session) StmtCommit() error {
 		st.doNotCommit = err
 		return err
 	}
+	if memTracker != nil {
+		memTracker.Consume(int64(st.Transaction.Size()))
```

Comment:
for situation like:
```
begin;
insert1
insert2
commit;
```
insert1 will add 1k to transaction buffer, insert2 will add 1k to transaction buffer.

so when insert2 step in here will +3k at here..

it's hard to track transaction memory in stmt level if one transaction have multiple statements. - -